backend/src/packages/chaiNNer_standard/__init__.py

```diff
@@ -1,8 +1,29 @@
 from sanic.log import logger
 
-from api import add_package
+from api import KB, MB, Dependency, add_package, is_arm_mac, is_windows
 
-package = add_package(__file__, name="chaiNNer_standard", dependencies=[])
+package = add_package(
+    __file__,
+    name="chaiNNer_standard",
+    description="The standard set of nodes for chaiNNer.",
+    dependencies=[
+        # Dependency("OpenCV", "opencv-python", "4.7.0.68", 30 * MB, import_name="cv2"),
```

Comment:
Comments should be removed. Though this code might be able to actually go there instead of using the hardcoded non-Dependency-class version